lib/components/dropdown.vue

```diff
@@ -28,7 +28,7 @@
             <span class="sr-only">{{toggleText}}</span>
         </b-button>
 
-        <div :class="['dropdown-menu',{'dropdown-menu-right': right}]"
+        <div :class="['dropdown-menu',{'dropdown-menu-right': right, show: visible}]"
```

Comment:
Original BS4-Beta:

![image](https://user-images.githubusercontent.com/5158436/29224407-74d02512-7edf-11e7-89ba-d0ee06bb532d.png)

But the outer show seems in unneeded! (Man we have to open many Issues in twbs repo :D) So you suggest removing one from the container?